gnovm/pkg/gnolang/store.go

```diff
@@ -798,14 +798,17 @@ func (ds *defaultStore) AddMemPackage(memPkg *std.MemPackage) {
 			bm.StopStore(size)
 		}()
 	}
-	memPkg.Validate() // NOTE: duplicate validation.
+	err := ValidateMemPackage(mpkg) // NOTE: duplicate validation.
```

Comment:
```suggestion
	err := SanitizeForStore(mpkg)
	err := ValidateMemPackage(mpkg) // NOTE: duplicate validation.
```

Perhaps we can implement the sanitization I mentioned earlier here; this would ensure that this validation is no longer considered a duplicate.